email-client/build.gradle

```diff
@@ -21,7 +21,9 @@ dependencies {
 
   annotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.6'
   compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.6'
-  
+  compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: '3.0.2'
+  compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: '3.0.2'
```

Comment:
Can we use the latest version for both of the above please? I think it is `5.0.1`

Reply:
done